doc/source/whatsnew/v1.3.0.rst

```diff
@@ -356,6 +356,7 @@ Reshaping
 - Bug in :func:`join` over :class:`MultiIndex` returned wrong result, when one of both indexes had only one level (:issue:`36909`)
 - :meth:`merge_asof` raises ``ValueError`` instead of cryptic ``TypeError`` in case of non-numerical merge columns (:issue:`29130`)
 - Bug in :meth:`DataFrame.join` not assigning values correctly when having :class:`MultiIndex` where at least one dimension is from dtype ``Categorical`` with non-alphabetically sorted categories (:issue:`38502`)
+- ``value_counts`` returns keys in original order (:issue:`12679`)
```

Comment:
``` :meth:`Series.value_counts` ```, can you add the other issue that is being closed here as well

Reply:
Done https://github.com/pandas-dev/pandas/pull/39009/commits/d85b1aa47fed1da8d92b3843bb2736816815aa9d